data/so-Osma

```diff
@@ -0,0 +1,16 @@
+---
+version: 2
+anglicized_name: Somali (Osmanya script)
+native_name: Soomaali (Farta Cismaanya)
+codepoints:
+- !ruby/range 66..68       # B,C,D
+- !ruby/range 70..72       # F,G,H
+- !ruby/range 74..78       # J,K,L,M,N
+- !ruby/range 81..84       # Q,R,S,T
+- !ruby/range 87..89       # W,X,Y
+- !ruby/range 98..100      # b,c,d
+- !ruby/range 102..104     # f,g,h
+- !ruby/range 106..110     # j,k,l,m,n
+- !ruby/range 113..116     # q,r,s,t
+- !ruby/range 119..121     # w,x,y
```

Comment:
Please remove these Latin ranges. (I'm sorry I wasn't clear in my earlier request about this.)

Reply:
Removed in https://github.com/typekit/speakeasy/pull/43/commits/f767ec078130c0d47b7a588681896c2fc2fdda2c.